app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -48,6 +48,8 @@ What you’ll learn will depend on your course and the qualifications it offers.
 
 Initial teacher training helps you meet the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards). You must meet the teachers’ standards to be awarded QTS.
 
+All initial teacher training courses cover [teaching pupils with SEND](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs). Once you've gained QTS, you can look for courses that specialise in SEND to develop your training further. 
```

Comment:
shouldn't we expand on SEND as the first reference on this page?

```suggestion
All initial teacher training courses cover [teaching pupils with special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs). Once you've gained QTS, you can look for courses that specialise in SEND to develop your training further. 
```

Reply:
I think the second sentence needs to be a bit clearer. I presume you are talking about NPQs, but the way it is phrased sounds like this is part of initial teacher training?

Also, I think the outcome of doing further training is not to `develop your training` but to specialise etc.?